Makefile

```diff
@@ -1,6 +1,7 @@
 init:
 ifeq ($(TRAVIS), true)
 		pip install -r reqs/travis-requirements.txt
+		pip list --local
```

Comment:
Listing the dependencies actually be used in the build can help debug dependency issues going forward.